src/Raven.Studio/wwwroot/App/views/manage/cluster.html

```diff
@@ -5,8 +5,13 @@
             <div class="flex-header flex-horizontal">
                 <button data-bind="click: addNode, enable: canAddNodes" class="btn btn-primary"><i class="icon-plus"></i> <span>Add Node to cluster</span></button>
                 <div class="flex-separator"></div>
-                <div class="flex-end text-muted" data-bind="if: licenseSummary">
-                    <small>License limits: <span data-bind="text: licenseSummary"></span></small>
+                <div class="flex-end text-muted" data-bind="if: utilizedCores() > 0">
+                    <small>Available resources: <strong data-bind="text: utilizedCores, css: cssCores" data-toggle="tooltip" title="Total Assigned CPU cores"></strong>
+                        / <strong data-bind="text: maxCores" data-toggle="tooltip" title="Licensed cores"></strong>
+                        <span data-bind="if: totalServersCores() > maxCores()">
+                            / <strong data-bind="text: totalServersCores" data-toggle="tooltip" title="Total number of available cores"></strong>
```

Comment:
title : total cores available where? in license on cluster? - maybe it is worth to make disambiguation on this

Reply:
done